functional_testing/Grid_API_Testing/api_testing/grid_apis/orchestrator_base.py

```diff
@@ -1,20 +1,10 @@
 from zeroos.orchestrator import client as apiclient
+from api_testing.grid_apis import JWT
 from testconfig import config
 
 class GridPyclientBase(object):
     def __init__(self):
         self.config = config['main']
```

Comment:
union configuration parsing steps in the same context. You can move this to the __init__ file